lib/Sema/CSStep.cpp

```diff
@@ -490,6 +490,27 @@ bool DisjunctionStep::shouldStopAt(const DisjunctionChoice &choice) const {
           shortCircuitDisjunctionAt(choice, lastChoice));
 }
 
+bool swift::isSIMDOperator(ValueDecl *value) {
+  if (!value)
+    return false;
+
+  auto func = dyn_cast<FuncDecl>(value);
+  if (!func)
```

Comment:
Nit: can combine two ifs into one `!(func && function->isOperator())`, and same could be done for nominal using De Morgan's Law :)

Reply:
I like breaking out the conditions here.